src/Adapter/MSTest.CoreAdapter/MSTest.CoreAdapter.csproj

```diff
@@ -111,6 +111,20 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
+    <None Include="Resources\xlf\Resource.cs.xlf" />
```

Comment:
1. Why these files were not earlier? Are we not supporting localization for resources present in core adapter?
2. In case we are not localizing resources by design, please remove these language specific resources.

Reply:
We were not supporting localization for resources in core adapter. Will remove language specific files.